.github/workflows/quality.yml

```
@@ -0,0 +1,36 @@
+name: Quality
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)